src/git-auth-helper.ts

```diff
@@ -159,10 +166,12 @@ class GitAuthHelper {
         )
       } else {
         // Configure HTTPS instead of SSH
-        await this.git.submoduleForeach(
-          `git config --local '${this.insteadOfKey}' '${this.insteadOfValue}'`,
-          this.settings.nestedSubmodules
-        )
+        for (const insteadOfValue of this.insteadOfValues) {
```

Comment:
when configuring auth, now we register multiple instead-of values

this translates submodule URLs like:

`git@github.com:my-org/my-submodule.git` -\> `https://github.com/my-org/my-submodule.git`
`org-123456@github.com:my-org/my-submodule.git` -\> `https://github.com/my-org/my-submodule.git`